app/[lang]/(board)/board/[slug]/[job_slug]/page.tsx

```diff
@@ -0,0 +1,488 @@
+"use client";
+import { getDictionary } from "@/app/[lang]/dictionaries";
+import { getListedJob, getListedJobs } from "@/lib/api/jobs";
+import { Job } from "@/types/job-schema";
+import { useRouter } from "next/navigation";
+import { useEffect, useRef, useState } from "react";
+import { FaAt, FaFacebook, FaGithub, FaInstagram, FaLink, FaLinkedin, FaPhone, FaTwitter } from "react-icons/fa";
```

Comment:
Use [lucide.dev](https://lucide.dev) icons instead, just as you did in line 10 with: 
```tsx
import { AlignEndHorizontal, ArrowLeft, Bookmark, MapPin, Share2 } from "lucide-react";
```
(If some icons from [lucide.dev](https://lucide.dev) are missing / not found in ts, you might need to wait for #22 to be merged first.)

Reply:
transfered into separate isuue